examples/CMakeLists.txt

```
@@ -110,7 +110,7 @@ macro(add_sdl_example_executable TARGET)
         target_link_libraries(${TARGET} PRIVATE GL)
     elseif(EMSCRIPTEN)
         set_property(TARGET ${TARGET} PROPERTY SUFFIX ".html")
-        target_link_options(${TARGET} PRIVATE -sALLOW_MEMORY_GROWTH=1)
+        target_link_options(${TARGET} PRIVATE -sALLOW_MEMORY_GROWTH=1 -s EXPORTED_RUNTIME_METHODS=[HEAPU8])
```

No space between `-s` and `EXPORT_RUNTIME_METHODS=[HEADU8]` to make sure CMake does not remove the `-s` if it sees multiple `-s`es.
```suggestion
        target_link_options(${TARGET} PRIVATE -sALLOW_MEMORY_GROWTH=1 -sEXPORTED_RUNTIME_METHODS=[HEAPU8])
```